benchexec/tools/mlb.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool):
```

Comment:
This is using our old and outdated API `BaseTool`. Please switch to `BaseTool2` and adjust your module accordingly. We have documented [hints](https://github.com/sosy-lab/benchexec/blob/main/doc/tool-integration.md#migrating-tool-info-modules-to-new-api) for this, and it should be not much effort.

Otherwise certain features will not work.